### PR TITLE
Create rule for 148apps

### PR DESCRIPTION
**Pull request by philip94**
Create 148apps.com.xml

### CODE REVIEW

File: src/chrome/content/rules/148apps.com.xml [hunk 1]
```diff
@@ -0,0 +1,25 @@
+<!--
+	Nonfunctional hosts in *.148apps.com:
+	    beta.images.148apps.com
+	Redirect to "www.148apps.com":
```
Comment (J0WI) [inline src/chrome/content/rules/148apps.com.xml]: You should add them anyway because a lot of resources are loaded over those domains.

File: src/chrome/content/rules/148apps.com.xml [hunk 2]
```diff
@@ -0,0 +1,25 @@
+<!--
+	Nonfunctional hosts in *.148apps.com:
+	    beta.images.148apps.com
```
Comment (J0WI) [inline src/chrome/content/rules/148apps.com.xml]: Please specify the type of error.

File: src/chrome/content/rules/148apps.com.xml [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+	    forum.148apps.com
+	    images.148apps.com
+    	    img.148apps.com
+	    media.148apps.com
+    	    beta.media.148apps.com
+	    static.148apps.com
+	    beta.static.148apps.com
+            store.148apps.com
+
+	Http redirect:
```
Comment (J0WI) [inline src/chrome/content/rules/148apps.com.xml]: Since you have this as an active target this comment is confusing.

File: src/chrome/content/rules/148apps.com.xml [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+	    www.148apps.com
+-->
+<ruleset name="148apps">
```
Comment (J0WI) [inline src/chrome/content/rules/148apps.com.xml]: The site has horrible mixed content issues, so please add the mixedcontent platform.

**Comment by philip94** [comment]
I updated the ruleset, hopefully it now contains the necessary changes.

File: src/chrome/content/rules/148apps.com.xml [hunk 5]
```diff
@@ -1,25 +1,23 @@
...
-	    images.148apps.com
-    	    img.148apps.com
-	    media.148apps.com
-    	    beta.media.148apps.com
-	    static.148apps.com
-	    beta.static.148apps.com
-            store.148apps.com
+	    beta.images.148apps.com (Error 403 - Forbidden)
+    	beta.media.148apps.com (Error 403 - Forbidden) 
+	    static.148apps.com (Error 404)
```
Comment (J0WI) [inline src/chrome/content/rules/148apps.com.xml]: Use `http://static.148apps.com/img/148apps-logo-266x.png` as test url

File: src/chrome/content/rules/148apps.com.xml [hunk 6]
```diff
@@ -1,25 +1,23 @@
 <!--
 	Nonfunctional hosts in *.148apps.com:
-	    beta.images.148apps.com
-	Redirect to "www.148apps.com":
-	    ads.148apps.com
-	    forum.148apps.com
-	    images.148apps.com
-    	    img.148apps.com
```
Comment (J0WI) [inline src/chrome/content/rules/148apps.com.xml]: Why did you remove `img.148apps.com` and `beta.static.148apps.com` completely?

**Comment by philip94** [comment]
Did the changes as you requested. "img" and "beta.static" were accidentally removed in the previous commit. Added them again.

**Comment by J0WI** [comment]
Thanks!

**Comment by philip94** [comment]
@J0WI The domain does not seem to be in the HTTPS Everywhere Atlas. Is there anything else that needs to be done in order the rule to take effect? I need to delete the branch or?

**Comment by J0WI** [comment]
I can find it in the atlas: https://www.eff.org/https-everywhere/atlas/domains/148apps.com.html
It may need some time to update. The rule will ship in the next release of HTTPS Everywhere.

**Comment by philip94** [comment]
Got it, thanks!
